db/migrate/20220713090517_create_reports.rb

```diff
@@ -0,0 +1,16 @@
+class CreateReports < ActiveRecord::Migration[6.0]
+  def change
+    create_table :reports do |t|
+      t.references :from_user, foreign_key: {to_table: :users}
+      t.references :to_user, foreign_key: {to_table: :users}
+      t.references :department, foreign_key: true
+      t.datetime :report_date
+      t.string :today_plan
```

Comment:
```suggestion
      t.text :today_plan
```
các trường today_plan, today_work, today_problem .. nên là trường text nhé